src/test/java/com/microsoft/aad/msal4j/TestConfiguration.java

```diff
@@ -63,7 +63,7 @@
 
 
     public static String INSTANCE_DISCOVERY_RESPONSE = "{" +
-            "\"tenant_discovery_endpoint\":\"https://login.microsoftonline.com/organizations/v2.0/.well-known/openid-configuration\"," +
+            "\"tenant_discovery_endpoint\":\"https://login.microsoftonline.com/organizations/v2.0/.well-known/openid-appConfiguration\"," +
```

Comment:
>app [](start = 117, length = 3)

why adding app?

Reply:
will fix - side effect of refactoring with IDE